migration-engine/connectors/sql-migration-connector/src/sql_schema_differ.rs

```diff
@@ -271,14 +271,16 @@ impl<'schema> SqlSchemaDiffer<'schema> {
     fn create_indexes(&self) -> Vec<CreateIndex> {
         let mut steps = Vec::new();
 
-        for table in self.created_tables() {
-            for index in &table.indices {
-                let create = CreateIndex {
-                    table: table.name.clone(),
-                    index: index.clone(),
-                };
-
-                steps.push(create)
+        if !self.sql_family.is_mysql() {
```

Comment:
nitpick (no need to change): I would find `if self.sql_family.is_mysql() { return Vec::new() }` a tiny bit more readable